doc/examples/run_returning_interaction.rb

```diff
@@ -0,0 +1,53 @@
+# Example: RunReturningInteraction
+# This is an example from within niiwin: https://github.com/animikii/niiwin-mvp/blob/main/app/controllers/i_app_data/i_widgets_controller/managed_by_niiwin.rb
+
+# app/controllers/i_app_data/i_widgets_controller/managed_by_niiwin.rb
+def create
+  authorize([:i_app_data, IWidget])
+  ar_attrs = params
+               .fetch(:i_widget, {})
+               .to_unsafe_hash
+               .transform_values(&:presence) # convert empty strings to nil for interaction input filters
+  @i_widget = ::IAppData::IWidgets::Create.run_returning!(
+    { i_widget: IWidget.new(ar_attrs) }.merge(ar_attrs),
+    )
+  if @i_widget.errors_any?
+    @nw_table = Niiwin::NwTable.find(:i_widget)
+    @ar_instance = @i_widget
+    render template: nw_new_template
+  else
+    redirect_to(i_app_data_i_widgets_path)
+  end
+end
+
+# The invoked interaction:
+# app/interactions/i_app_data/i_widgets/create.rb
+module IAppData
+  module IWidgets
+    # Creates a new IWidget
+    class Create < ReturningInteraction
+
+      returning :i_widget
+
+      record :i_widget, class: IWidget, default: -> { IWidget.new }
```

Comment:
```suggestion
      record :i_widget, class: IWidget
```
Let's remove the default value. This is correct, however, I don't think it is supportive of clarity.